bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/WZTo1L3Nu01j_5f_NLO_FXFX/WZTo1L3Nu01j_5f_NLO_FXFX_madspin_card.dat

```diff
@@ -0,0 +1,30 @@
+#************************************************************
+#*                        MadSpin                           *
+#*                                                          *
+#*    P. Artoisenet, R. Frederix, R. Rietkerk, O. Mattelaer * 
+#*                                                          *
+#*    Part of the MadGraph5_aMC@NLO Framework:              *
+#*    The MadGraph5_aMC@NLO Development Team - Find us at   *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#Some options (uncomment to apply)
+#
+#directory for gridpack mode
+set ms_dir ./madspingrid
+
+#initialization parameters
+set Nevents_for_max_weigth 250 # number of events for the estimate of the max. weight
+set max_weight_ps_point 400  # number of PS to estimate the maximum for each event
+
+# BW_cut = 40 required by Hbb
+set BW_cut 40                # cut on how far the particle can be off-shell
+ 
+#to properly limit the number of concurrent processes for grid running
+set max_running_process 1
+
+decay w- > ta- vt~
```

Comment:
is the madspin_card really needed?

Reply:
same as https://github.com/cms-sw/genproductions/pull/1449#issuecomment-339350169 ?

Reply:
@kdlong can you please confirm that is valid also here?

Reply:
The MadSpin card isn't "needed" but it was included in the past to trigger the inclusion of
some spin-related info in the LHE file to be passed to Pythia to be used for the tau
decays. 

There is now an additional syntax for this, which you can see the new syntax in this
card:

https://github.com/cms-sw/genproductions/blob/pre2017/bin/MadGraph5_aMCatNLO/cards/production/13TeV/WZTo3LNu01j_5f_NLO_FXFX/WZTo3LNu01j_5f_NLO_FXFX_madspin_card.dat#L32

If you leave it as is, it will fail at the MadSpin step (right?)